.github/workflows/template-web.yml

```diff
@@ -70,7 +70,20 @@ jobs:
         with:
           azPSVersion: "3.1.0"
           inlineScript: |
-            az deployment group create -n ghaction -g ${{ secrets.AZURE_RESOURCE_GROUP_NAME }} --template-file deploy/Web/web.bicep --parameters webAppName=${{ secrets.WEBAPP_NAME }} servicePlanName=${{ secrets.SERVICE_PLAN_NAME }} servicePlanSku=${{ secrets.SERVICE_PLAN_SKU }} hubWebAppName=${{ secrets.HUB_WEBAPP_NAME }} apiName=${{secrets.API_RESOURCE_NAME}} updaterName=${{secrets.UPDATER_RESOURCE_NAME}}
+            $deploymentName = 'ghaction${{ github.sha }}'
+            $resourceGroupName = '${{ secrets.AZURE_RESOURCE_GROUP_NAME }}'            
+            $webAppName = '${{ secrets.WEBAPP_NAME }}'
+            $servicePlanName = '${{ secrets.SERVICE_PLAN_NAME }}'
+            $servicePlanSku = '${{ secrets.SERVICE_PLAN_SKU }}'
+            $hubWebAppName = '${{ secrets.HUB_WEBAPP_NAME }}'
+            $apiName = '${{ secrets.API_RESOURCE_NAME }}'
+           
+            az deployment group create -n "$($deploymentName)" `
+               --resource-group  "$($resourceGroupName)" `
+               --template-file deploy/Web/web.bicep `
+               --parameters webAppName="$($webAppName)" servicePlanName="$($servicePlanName)" servicePlanSku="$($servicePlanSku)" hubWebAppName="$($hubWebAppName)" apiName="$($apiName)" 
+               
+          #  az deployment group create -n ghaction -g ${{ secrets.AZURE_RESOURCE_GROUP_NAME }} --template-file deploy/Web/web.bicep --parameters webAppName=${{ secrets.WEBAPP_NAME }} servicePlanName=${{ secrets.SERVICE_PLAN_NAME }} servicePlanSku=${{ secrets.SERVICE_PLAN_SKU }} hubWebAppName=${{ secrets.HUB_WEBAPP_NAME }} apiName=${{secrets.API_RESOURCE_NAME}} updaterName=${{secrets.UPDATER_RESOURCE_NAME}}    
```

Comment:
can we remove this commented out script?

Reply:
Yes, I forgot to do it.
Wait a minute

Reply:
@MarcusFelling  done

Reply:
LGTM! Thanks, @Soruk!